gammapy/modeling/models/spatial.py

```diff
@@ -57,6 +94,11 @@ def to_dict(self):
         data["parameters"] = data.pop("parameters")
         return data
 
+    @abc.abstractmethod
```

Comment:
Suggest to delete this abstractmethod in the base class. The method `to_region` isn't there e.g. for the SpatialConstant model, and then linters show warnings when sub-classes don't implement the ABC interface.
The normal solution in Python is to just not use ABC, i.e. delete this abstract method in the base class.